ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+[Before submitting an issue, please check that your isssue hasn't been already filed]
+
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Expected behavior
+
+[What you expect to happen]
+
+### Actual behavior
+
+[What actually happens]
+
+### Reproducibility
+
+[What percentage of the time does it reproduce?]  
+
+### Versions
+
+[If a tagged version, you can get this information by inspecting the `ITK_VERSION_MAJOR` `ITK_VERSION_MINOR` and `ITK_VERSION_PATCH` variable values in the `ITKConfig.cmake` file]
+
+[If the commit number is required, run `$ git rev-parse --short HEAD`]
+
+### Platform
+
+[Which your OS and compiler are]
+
+
+### Additional Information
+
+[Any additional information, configuration or data that might be necessary to reproduce the issue.]
+
+### Label
+
+[Choose the label that best fits the issue]
+
+### Milestones
```

Comment:
Give a URL of where to find the list of Milestones.

Reply:
Same comment as for the labels.